.github/workflows/e2e-tests.yml

```diff
@@ -35,30 +39,56 @@ jobs:
 
       - name: Capture Deployment URL
         run: |
-          DEPLOYMENT_URL=${{ github.event.deployment_status.environment_url }}
-          echo "DEPLOYMENT_URL = $DEPLOYMENT_URL"
-          echo "BASE_URL=$DEPLOYMENT_URL" >> $GITHUB_ENV
+          if [[ "${{ github.event_name }}" == "schedule" ]]; then
+            # Use production URL for scheduled runs
+            echo "BASE_URL=https://goodparty.org" >> $GITHUB_ENV
+            echo "SKIP_TESTS=false" >> $GITHUB_ENV
+          else
+            DEPLOYMENT_URL=${{ github.event.deployment_status.environment_url }}
+            if [[ "$DEPLOYMENT_URL" == *"-styleguide"* ]]; then
+              echo "Skipping tests for styleguide deployment"
+              echo "SKIP_TESTS=true" >> $GITHUB_ENV
+            else
+              echo "DEPLOYMENT_URL = $DEPLOYMENT_URL"
+              echo "BASE_URL=$DEPLOYMENT_URL" >> $GITHUB_ENV
+              echo "SKIP_TESTS=false" >> $GITHUB_ENV
```

Comment:
I tried this approach.. the problem w/ it is that GitHub still sees, and reports in the PR UI, this as a "run" of a task, so  makes things really confusing.  So I think we need to find out why the lines `6-7` aren't being honored instead of trying to do it this way.